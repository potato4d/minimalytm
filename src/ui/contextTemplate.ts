import { App, MenuItemConstructorOptions } from 'electron'

export function createContextTemplate(app: App): MenuItemConstructorOptions[] {
  return [
    {
      label: 'Quit',
      accelerator: 'Command+Q',
      click: () => {
        app.quit()
      }
    }
  ]
}
