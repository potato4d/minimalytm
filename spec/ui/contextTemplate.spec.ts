import { createContextTemplate } from '../../src/ui/contextTemplate'

describe('contextTemplate', () => {
  test('createContextTemplate()', () => {
    expect(createContextTemplate({} as any)).toMatchSnapshot()
  })
})
