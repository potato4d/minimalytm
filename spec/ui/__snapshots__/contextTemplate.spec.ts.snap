// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contextTemplate createContextTemplate() 1`] = `
Array [
  Object {
    "accelerator": "Command+Q",
    "click": [Function],
    "label": "Quit",
  },
]
`;
